packaging/fedora/snapd.spec

```diff
@@ -426,6 +426,8 @@ install -m 644 -D data/info %{buildroot}%{_libexecdir}/snapd/info
 
 # Install bash completion for "snap"
 install -m 644 -D data/completion/snap %{buildroot}%{_datadir}/bash-completion/completions/snap
+install -m 644 -D data/completion/complete.sh %{buildroot}%{_libexecdir}/snapd
+install -m 644 -D data/completion/etelpmoc.sh %{buildroot}%{_libexecdir}/snapd
```

Comment:
@Conan-Kudo Note that those are special files that make snapd implement completion internally. The things you are thinking about about are in the usual location.